ops-bedrock/docker-compose.yml

```diff
@@ -197,6 +202,36 @@ services:
       OP_CHALLENGER_HD_PATH: "m/44'/60'/0'/0/4"
       OP_CHALLENGER_NUM_CONFIRMATIONS: 1
 
+  da-server:
+    image: us-docker.pkg.dev/oplabs-tools-artifacts/images/da-server:devnet
+    build:
+      context: ../
+      dockerfile: ./op-plasma/Dockerfile
+    command: >
+      da-server
+      --file.path=/data
+      --addr=0.0.0.0
+      --port=3100
+      --log.level=debug
+    ports:
+      - "3100:3100"
+    volumes:
+      - "l2_data:/data"
+
+  sentinel:
+    image: quarry/sentinel # TODO(10141): We need a public image for this (sentinel is out of repo)
+    volumes:
+      - "${PWD}/../.devnet/rollup.json:/rollup.json"
+    # private-key is mnemonic index 5 of the test mnemonic.
```

Comment:
Are we able to pass the mnemonic like we do for `op-challenger`? https://github.com/ethereum-optimism/optimism/blob/f2c6debe7a575d49deaef0b1635bf3f4dc70901b/ops-bedrock/docker-compose.yml#L198-L199


Not a big deal if it's not supported but more clearly a test key if we can.

Reply:
Unfortunately not supported - sentinel isn't our piece of software